src/main/java/Duke.java

```diff
@@ -1,10 +1,205 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    public static final String BYE_COMMAND = "bye";
+    public static final String CHANGE_COMMAND = "change";
+    public static final String LANG_COMMAND = "lang";
+    public static final String LIST_COMMAND = "list";
+    public static final String MARK_COMMAND = "mark";
+    public static final String UNMARK_COMMAND = "unmark";
+    public static final String TODO_COMMAND = "todo";
+    public static final String DEADLINE_COMMAND = "deadline";
+    public static final String DEADLINE_BY = "/by";
+    public static final String EVENT_COMMAND = "event";
+    public static final String EVENT_START_FROM = "/from";
+    public static final String EVENT_END_TO = "/to";
+    /** A fixed sized buffer to trim strings at the right index */
+    public static final int COMMAND_BUFFER = 1;
```

Comment:
Constant variables are all in capital letters which adheres to the Java Coding standard